tests/conftest.py

```diff
@@ -4,15 +4,20 @@
 import os
 import sys
 import logging
-from shutil import rmtree
 
 
 def pytest_addoption(parser):
     parser.addoption(
-        "--skip_end_to_end",
+        "--skip_slow",
```

Comment:
Question: can you please explain this flag? Does it always skip all tests that are slow? If so, how/when do we test the slow ones?

Reply:
I will document this ASAP !